tests/test_credits.py

```diff
@@ -1,21 +1,24 @@
-import pytest
-import os
-from credit import load_credits, logger
+import pytest, os, json
+from main import logger
+from credit import load_credits
 from unittest import mock
 
 
 @pytest.mark.parametrize(
-    ('path', 'expected_str'),
+    ('error','path', 'expected_str'),
     (
-        (os.path.dirname(os.path.dirname(os.path.realpath(__file__))) + '/credential.txt', 'to-do-app/credential.txt'),
-        (os.path.dirname(os.path.realpath(__file__)) + '/crap.txt', 'data in credentials.txt is not json'),
+        (FileNotFoundError, os.path.dirname(os.path.dirname(os.path.realpath(__file__))) + '/credential.txt', 'to-do-app/credential.txt'),
+        (TypeError, os.path.dirname(os.path.realpath(__file__)) + '/crap.txt', 'data in credentials.txt is not json'),
+        (TypeError, os.path.dirname(os.path.realpath(__file__)) + '/not_dict.txt', 'Data in credentials.txt is not dictionary'),
+        (TypeError, os.path.dirname(os.path.realpath(__file__)) + '/not_string.txt', '2 keys and 2 values in credit dictionary are not string')
     ),
-    ids = ['No credentials.txt case', 'Data in file is not json']
+    ids = ['No credentials.txt case', 'Data in file is not json', 'Data is not dictionary', 'Credentials have not string data']
 )
-def test_load_credits(path, expected_str):
-    with mock.patch.object(logger, 'error') as mock_error:
-        load_credits(path)
-        assert expected_str in mock_error.call_args[0][0]
+def test_load_credits(error, path, expected_str):
```

Comment:
Please add a case for a normal case where data is actually loaded.

Reply:
Done.